homeassistant/components/fritzbox/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "title": "AVM FRITZ!Box",
+    "flow_title": "AVM FRITZ!Box: {name}",
+    "step": {
+      "user": {
+        "title": "AVM FRITZ!Box",
+        "description": "Enter your AVM FRITZ!Box information.",
+        "data": {
+            "host": "Host or IP address",
+            "username": "Username",
+            "password": "Password"
+        }
+      },
+      "confirm": {
+        "title": "AVM FRITZ!Box",
+        "description": "Do you want to set up {name}? Manual configurations for this device will be overwritten.",
```

Comment:
When will the manual entry be overwritten?

Reply:
I've copied this text from Samsung TV - we had there a discussion about it. When the import step failed (e.g. device offline) and starts a flow via ssdp, the configured values in the yaml will never found their way into HA because of the already configured IP.

Reply:
I'd specify that it's config yaml entries we're referring to.